puppet/zulip/files/nginx/zulip-include-app.d/keepalive-loadbalancer.conf

```diff
@@ -0,0 +1,17 @@
+# This file should be used if the nginx is behind a dedicated
+# loadbalancer.  This allows nginx to be configured for long
+# keep-alive timeouts, to keep connections open to the loadbalancer.
+#
+# All timeouts here should be _longer_ than those on the loadbalancer.
+# This ensures that there cannot be a race condition between nginx
+# deciding to shut down the connection, and the load-balancer sending
+# a request, resulting in a 502 from the loadbalancer.
+#
+# The timeouts on the loadbalancer are assumed to be 20min (1200s);
+# these timeouts are hence for 21min.
+
+keepalive_timeout     1260;
+client_header_timeout 1260;
+client_body_timeout   1260;
+
+keepalive_requests     500;
```

Comment:
How did you pick this number?  http://nginx.org/en/docs/http/ngx_http_core_module.html#keepalive_requests suggests it's the number per-request, and it seems pretty unlikely a single request would hit 500 keepalives with our model (where the only longpolling has an application-level timeout of about a minute, which we might in the future increase to something as high as 20 minutes).

Reply:
Picked to be large-ish and "100 or higher," per ALB recommendation.  I don't think it will have a significant effect, since as you note we're unlikely to hit it because of the number of longpoll requests.